### ISSUE TITLE
streamlining the editing process

### ISSUE THREAD

**Issue by geextahslex** [nptr/msstyleEditor#131]
**Is your feature request related to a problem? Please describe.**
Problem is that you have to go trough hundreds of options, when all you want is to set font color to "A" and set background color to "B"

**Describe the solution you'd like**
Something like an "easy-mode" where all font colors, and background colors are linked to one simplified window.
An interface like "Dark background and light text" addon for firefox

![example](https://github.com/nptr/msstyleEditor/assets/139760266/33980844-4ac3-4cad-a5f3-ece17a53a2f4)

**Describe alternatives you've considered**
none

**Additional context**
As we all know, everytime there is a major windows update the old themes brakes somewhere. So having an easier way to set up your desired theme would be more user friendly. As you can't make one theme and roll with it forever.

Thank you

**Comment by nptr** [comment]
There is the plan to export and import sets of properties. With this you could then create patches like "light fonts" , "dark fonts", ... and achieve streamlining this way.

It's an open task still. See #77 and #104.

**Comment by geextahslex** [comment]
It would be just a relief to setup a base theme (no fancy theme) without going trough hundreds of options. I atempted to do a custom theme but everytime I saw this huge list of variables I felt like "okay, maybe next time" ^^

**Comment by nptr** [comment]
I agree, it is hard to make a custom themes with msstyleEditor. For that, there is a better tool. The original goal of mssyleEditor was to modify / fix custom themes I didn't fully like. From there it grew and is now somewhere in between.
